Packs/FiltersAndTransformers/ReleaseNotes/1_2_19.md

```diff
@@ -0,0 +1,13 @@
+
+#### Scripts
+
+##### LastArrayElement
+- Updated the Docker image to: *demisto/python3:3.10.11.61265*.
+
+- Fixed an issue with handling null value given to the argument parameters. 
+
+##### FirstArrayElement
+- Updated the Docker image to: *demisto/python3:3.10.11.61265*.
+
+- Fixed an issue with handling null value given to the argument parameters. 
```

Comment:
```suggestion
- Fixed the way in which the script handled null value given to the argument parameters, instead of returning 'None', it will return a meaningful error message. 
```